src/devices/bus/nes/nes_slot.h

```diff
@@ -171,6 +170,12 @@ enum
 #define PPU_MIRROR_4SCREEN  5
 
 
+#define MMC1   0
+#define MMC1A  1
+#define MMC1B  2
+#define MMC1C  3
```

Comment:
Please avoid macros in headers when they aren’t really necessary.  Consider making this a scoped enum, and consider putting it inside a class where it’s relevant.

Reply:
Yeah, I should have figured this went against efforts to modernize the code. Scoped enum now in the all-but-the-kitchen-sink base class for all the cart types.